www/tenfourfox/Portfile

```diff
@@ -25,17 +26,30 @@ if {${subport} eq "${name}"} {
     checksums           rmd160  e9a499d3acfe69c622eeafc7ea68ef957b7bdf3c \
                         sha256  b5aef01b6d031e4b582460955e3a3b95f7b3bfb994cdfa0171f0cd6279ca21ce \
                         size    325688845
+    set appname TenFourFox
 
 } elseif {${subport} eq "${name}-devel"} {
     conflicts           ${name}
     description         Clone of Firefox, compatible with macOS 10.4+. Tracks latest commit on GitHub.
     long_description    ${description}
 
-    github.setup        classilla tenfourfox 656875dfebe724c31896e3073780638e2c213f81
-    version             20230921-FPR32
-    checksums           rmd160  82d5229d46ab25ec9850754d97fd4f7ccad8c5e3 \
-                        sha256  a523c6a9985f1df83c167839b36f8c3ed6a67e133baf42859cf0d826e3d17bda \
-                        size    326149990
+    github.setup        classilla tenfourfox cdc8b5a989ce0b8ced48edbf8166d460c67a6a0d
+    version             20240505-FPR32
+    checksums           rmd160  3ad5aa53b5d127c8d46da03cb9e94e5573193db8 \
+                        sha256  2a9bf00d8b535cc78111f1f9ece6c185aeac3635b08dad09c9efb082356d3631 \
+                        size    326144707
+    set appname TenFourFox
+
+} elseif {${subport} eq "aquafox"} {
+    description         TenFourFox fork with extra optimizations.
+    long_description    ${description}
+
+    github.setup        BlackBirdLC Aquafox 0785667d630acf269af554f538c19da181e5253f
```

Comment:
this is not a "-devel" port so you should use an upstream-tagged / released version.

Reply:
@BlackBirdLC This is equivalent to v. 1.0.0, right? Then just use that instead of the hash (author repo 1.0.0 v).

Reply:
@barracuda156 It is with the exception of the nostrip variants of mozcfg, which the tenfourfox Portfile expects to be present.

Reply:
@BlackBirdLC Got it. Maybe then just add https://github.com/BlackBirdLC/Aquafox/commit/0785667d630acf269af554f538c19da181e5253f commit as a patch on top of v 1.0.0?

Reply:
@barracuda156 I could create a new release named v1.0.0-0785667 without any binaries attached, or would that be considered cheating?

Reply:
That would cost me another 3 hrs of compilation with strictly identical result LOL

If you don’t mind, I’t rather prefer keeping 1.0.0 version, just with the patch added, so that we have the same source as before.
Inside `aquafox` subport block add something like:
```
patchfiles-append 0785667d630acf269af554f538c19da181e5253f.patch
```
(Add place the patch inside `files` directory.)